NomadAudio/src/Linux/LinuxDriverRegistry.cpp

```
@@ -0,0 +1,20 @@
+#include "../../include/AudioPlatformRegistry.h"
+#include "RtAudioDriver.h"
+#include <vector>
+#include <string>
+
+namespace NomadAudio {
+
+void RegisterPlatformDrivers(AudioDeviceManager& manager) {
+    manager.addDriver(std::make_unique<RtAudioDriver>());
+}
+
+PlatformAudioInfo GetPlatformAudioInfo() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)